homeassistant/components/griddy/sensor.py

```diff
@@ -0,0 +1,75 @@
+"""Support for August sensors."""
+import logging
+
+from homeassistant.components.sensor import DEVICE_CLASS_POWER
+from homeassistant.helpers.entity import Entity
+
+from .const import CONF_LOADZONE, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(hass, config_entry, async_add_entities):
+    """Set up the August sensors."""
+    coordinator = hass.data[DOMAIN][config_entry.entry_id]
+
+    settlement_point = config_entry.data[CONF_LOADZONE]
+
+    async_add_entities([GriddyPriceSensor(settlement_point, coordinator)], True)
+
+
+class GriddyPriceSensor(Entity):
+    """Representation of an August sensor."""
+
+    def __init__(self, settlement_point, coordinator):
+        """Initialize the sensor."""
+        self._coordinator = coordinator
+        self._settlement_point = settlement_point
+
+    @property
+    def unit_of_measurement(self):
+        """Return the unit of measurement."""
+        return "¢/kwh"
+
+    @property
+    def device_class(self):
+        """Return the class of this device, from component DEVICE_CLASSES."""
+        return DEVICE_CLASS_POWER
```

Comment:
Device classes require that the entity use its associated unit of measurement. This sensor does not do that. Docs: https://developers.home-assistant.io/docs/entity_sensor#available-device-classes

Reply:
removed